lib/auth/auth.go

```diff
@@ -843,11 +843,28 @@ func (a *Server) CheckU2FSignResponse(user string, response *u2f.AuthenticateCha
 		return trace.Wrap(err)
 	}
 
-	return u2f.AuthenticateVerify(u2f.AuthenticateVerifyParams{
-		Resp:       *response,
-		StorageKey: user,
-		Storage:    a.Identity,
-	})
+	ctx := context.TODO()
```

Comment:
Nit: Since you're here anyway, would it make sense to refactor this method to support passing a context?

Reply:
Propagated one layer up.
Going further will cause many more diffs and this PR is already too big

Reply:
That's fair.